src/module/Ui/templates/entity/view/partials/actions/trash.twig

```diff
@@ -8,4 +8,13 @@
         {% endset %}
         {% include 'common/csrfform' with {'form': uuidForm().getTrashForm(entity.getId()), 'sendbutton': trashbutton} %}
     </li>
-{% endif %}
\ No newline at end of file
+{% elseif entity.getTrashed() and isGranted('entity.restore', entity) %}
+    <li>
+        <a rel="nofollow" href="{{ url('uuid/restore', {'id' : entity.getId()}) }}">
```

Comment:
What's the reason that the `restore` action is an `a` while the (already existing) `trash` action is a `button`?

Reply:
The trash button uses a CSRF-Form, and thus is a button (submitting the form). Restore is atm not csrf secured, so it's just a link. Don't remember why though. I think I copied this from the link actions: https://github.com/serlo-org/athene2/blob/112387b5f93c7e668edef31bbe3f7515c439b9ea/src/module/Ui/templates/entity/view/partials/actions/link/add.twig#L17-L26